libr/core/panels.c

```diff
@@ -1910,17 +1910,27 @@ static int openMenuCb (void *user) {
 	return 0;
 }
 
-static void addMenu(RPanelsMenuItem *parent, const char *name, RPanelsMenuCallback cb) {
+static void addMenu(RCore *core, const char *parent, const char *name, RPanelsMenuCallback cb) {
+	RPanels *panels = core->panels;
 	RPanelsMenuItem *item = R_NEW0 (RPanelsMenuItem);
+	RPanelsMenuItem *p_item;
+	if (parent) {
+		ut64 addr = r_num_math (core->num, sdb_ht_find (panels->mht, parent, NULL));
+		p_item = (RPanelsMenuItem *)addr;
+		sdb_ht_insert (panels->mht, sdb_fmt ("%s.%s", parent, name), sdb_fmt ("%p", item));
```

Comment:
sorry the naming of the api/data structures is really confusing... you should use the `ht_*` API defined in `sdb/src/ht.h`, otherwise you'll end up using sdb anyway :) Sorry about it, anyway, at least you know now!